frontend/src/components/PlanHome/PlanHome.tsx

```diff
@@ -0,0 +1,152 @@
+import styles from './PlanHome.module.css'
+import editButton from '../../assets/editButton.png'
+import addMemberButton from '../../assets/addMemberButton.png'
+import { useParams } from 'react-router-dom'
+import { fetchPlanData } from '../Plan/hook/fetchPlanData'
+import { loadingStatus } from '../../utils/loadingStatus'
+import { fakeUserData } from '../PlanForm'
+import { MemberList } from './MemberList'
+import { PlanStage } from '../../utils/types'
+
+import { buildStyles, CircularProgressbar } from 'react-circular-progressbar';
+import 'react-circular-progressbar/dist/styles.css';
+
+type planTaskCompleted = number;
+type planPercentage = number;
```

Comment:
Lmk if this is good to do. I am trying make my code more readable lol

Reply:
2 nits: 

1) typings typically will be Title cased so these should be

`PlanTaskCompleted` and `PlanPercentage`

2) the naming of variables should give insight into its typing. `PlanTaskCompleted` isn't really clear, so maybe it could be like`NumPlanTasksCompleted` and similarly, `PlanPercentage` can be updated to `PercentPlanFunded` or similar